packages/grid/x-data-grid-pro/src/tests/rowEditing.DataGridPro.old.test.tsx

```diff
@@ -323,8 +335,9 @@ describe('<DataGridPro /> - Row Editing', () => {
     fireEvent.change(input, { target: { value: 'Adidas' } });
     clock.tick(500);
     fireEvent.keyDown(input, { key: 'Enter' });
+    await act(() => Promise.resolve());
 
-    await new Promise((resolve) => nativeSetTimeout(resolve)); // Wait for promise
+    await Promise.resolve(); // Wait for promise
```

Comment:
Should this be also wrapped in an `act`? There is one more case couple of lines below.

Reply:
I removed the second one. It was doing nothing.